app/views/help/index.html.erb

```diff
@@ -8,28 +8,89 @@
-      <%= render "govuk_publishing_components/components/list", {
+      <%= render "govuk_publishing_components/components/document_list", {
+        equal_item_spacing: true,
         items: [
-          sanitize("<a href='/help/about-govuk' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.about') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.about_description') + "</p>"),
-          sanitize("<a href='/help/accessibility' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.accessibility') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.accessibility_description') + "</p>"),
-          sanitize("<a href='/help/beta' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.beta') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.beta_description') + "</p>"),
-          sanitize("<a href='/help/browsers' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.browsers') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.browsers_description') + "</p>"),
-          sanitize("<a href='/help/cookies' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.cookies') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.cookies_description') + "</p>"),
-          sanitize("<a href='/help/update-email-notifications' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.email') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.email_description') + "</p>"),
-          sanitize("<a href='/help/privacy-notice' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.privacy_notice') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.privacy_notice_description') + "</p>"),
-          sanitize("<a href='/help/report-vulnerability' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.vulnerability') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.vulnerability_description') + "</p>"),
-          sanitize("<a href='/help/reuse-govuk-content' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.reuse_content') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.reuse_content_description') + "</p>"),
-          sanitize("<a href='/help/terms-conditions' class='govuk-link govuk-!-font-weight-bold'>" + t('help.index.terms') + "</a>
-          <p class='govuk-body govuk-!-margin-top-1'>" + t('help.index.terms_description') + "</p>")
+          {
+            link: {
+              text: t('help.index.about'),
+              path: "/help/about-govuk",
+              description: t('help.index.about_description'),
+              full_size_description: true
+            }
+          },
+          {
+            link: {
+              text: t('help.index.accessibility'),
+              path: "'/help/accessibility",
```

Comment:
It looks like the path here should be updated from `path: "'/help/accessibility"` to `path: "/help/accessibility"`